ambry-protocol/src/test/java/com/github/ambry/protocol/RequestResponseTest.java

```diff
@@ -369,14 +364,12 @@ public void putRequestResponseTest() throws IOException {
         blob, blobSize, blobKey);
 
     // Response test
-    for (boolean useByteBufContent : new boolean[]{true, false}) {
```

Comment:
Changes in this file are some cleanup and not related to user metadata change. (there was a travis failure, although this doesn't really fix the intermittent test error, it removes redundant code)